Documentation/Features/Software Requirements Specification.md

```diff
@@ -0,0 +1,255 @@
+
+[TOC]
+
+## 1. Introduction
+
+### 1.1 Purpose
+
+The reason we're developing this project is for learning purposes and to know how software is developed and delivered. 
+
+### 1.2 Intended Audience
+
+People that wants to share their knowledge and people who wants to learn.
+
+### 1.3 Intended Use
+
+The use of this document will include software developers, testers and teachers. The document is written as a reference to understand the requirements of this project. It will also be used to gap a bridge between developers, teachers and testers. 
+
+### 1.4 Scope
+
+Knowledge Net (KNet) is an application that matches Users that have a skill (teachers) with other users that want to learn that specific skill (students). Users will be able to create an account. During account creation, the users will specify which skills they are interested to teach/learn. The users can choose how they want to meet, online or on specified locations. The goal of the project is to create simple way for people to connect and a network for people to meet and learn. The users will be able to advertise what skills they want to teach or learn.
+
+* KNet is limited to being a skill sharing networking site. It will not supply the means to host a tutoring platform.
+* KNet is not intended to be a social media platform, and therefore contact between users is defined by the users themselves and not by KNet.
+* KNet is a non-profit organization and will not handle payments between users.
+* KNet will accept donations from users.
+
+  
+
+## 2. Overall Description
+
+### 2.1 User Needs
+
+The application will have two different types of users; teachers and students. The teachers must be able to advertise what skills they are able to teach. The students must be able to explore the site for advertised skills that they want to learn. The students can advertise for skills that they want to learn, so that they can find tutors for that skill. 
+
+### 2.2 Assumptions and Dependencies
+
+Everyone that will use KNet must have access to Internet. We are dependent on users that want to teach their skill. We assume that the people that use our portal wants to teach and learn. We don't host any platform for the users to teach and learn, therefore we are dependent on external platforms for the users to connect or meet for tutoring purposes. 
+
+## 3. System Features and Requirements
+
+In this section are all the functional requirements for the project.
+
+###       3.1 Functional Requirements
+
+###       3.2 External Interface Requirements
+
+**User:**
+
+* Device with an Internet browser. 
+
+* Internet connection. 
+
+  
+
+**KNet requirements:** 
+
+* Hosting environment (Cloud service) 
+
+* Docker 
+
+* IDE For coding 
+
+* Continuous Integration pipeline 
+
+* Continuous Delivery pipeline 
+
+* GitHub repository
+
+* MSSQL-database
+
+  
+
+###       3.3 System Features
+
+#### 3.3.1 Account
+
+```gherkin
```

Comment:
Might be enough to have system features only in Jira so you don't have to update both Github- and Jira documentation when you want to make a change.